spec/support/Cohortmanager.spec.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-unused-vars */
```

Comment:
I imagine you put this here when trying to commit. Eslint flagging stuff does get annoying trying to commit stuff. However, it's probably likely that the variable you were including in a commit wouldn't have been relevant to the commit. I'd say it's best not to ignore warnings as best as you are able to.

Reply:
I agree with Eduard here. If it's warning you about an unused variable, remove the variable instead of suppressing the warning